distod/src/test/scala/com/github/codelionx/distod/actors/partitionMgmt/JobChainerSpec.scala

```diff
@@ -0,0 +1,61 @@
+package com.github.codelionx.distod.actors.partitionMgmt
+
+import com.github.codelionx.distod.Settings.PartitionCompactionSettings
+import com.github.codelionx.distod.partitions.{FullPartition, StrippedPartition}
+import com.github.codelionx.distod.types.CandidateSet
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.concurrent.duration.FiniteDuration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/CodeLionX/distod/pullRequest?prid=5011078)